File: codeforces/69/A.cpp
#include<bits/stdc++.h>
#define endl       '\n'
#define pb         push_back
#define mod        1000000007
#define int        long long int
#define hii        cout<<"yes\n";
#define all(x)     x.begin(),x.end()
#define deb(x)     cout<<#x<<" : "<<x<<"\n";
#define FASTIO     ios_base::sync_with_stdio(0);cin.tie(0);cout.tie(0)

using namespace std;

void solve() {
	int n;
	cin >> n;
	int sum_x = 0, sum_y = 0, sum_z = 0;
	int x,y,z;
	
	for(int i = 0 ; i < n; ++i)
	{
		cin >> x >> y >> z;
		sum_x += x;
		sum_y += y;
		sum_z += z;
	}
	if(sum_x == 0 && sum_y == 0 && sum_z == 0)
		cout<<"YES"<<endl;
	else
		cout<<"NO"<<endl;
	
}

int32_t main() {
	FASTIO;/*
#ifndef ONLINE_JUDGE
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
#endif*/
	int t = 1;
	//cin >> t;
	while (t--) {
		solve();
	}
	return 0;
}

File: codeforces/69/A.java
import java.util.*;
public class Main {
    public static void main(String args[]) {
        
        Scanner sc = new Scanner(System.in);
        int test = sc.nextInt();
        int[] x = new int[test];
        int[] y = new int[test];
        int[] z = new int[test];
        int sumx = 0 , sumy = 0 , sumz = 0;
        
        for(int i = 0; i < test ; ++i)
        {
            x[i] = sc.nextInt();
            sumx = sumx + x[i];
            y[i] = sc.nextInt();
            sumy = sumy + x[i];
            z[i] = sc.nextInt();
            sumz = sumz + z[i];
        }
        
        if((sumx == 0) && (sumy == 0) && (sumz == 0))
        System.out.println("YES");
        else
        System.out.println("NO");
        
    }
}